build.xml

```diff
@@ -523,10 +431,7 @@
             description="Default build target">
         <zip zipfile="${dist}/picard-tools-${picard-version}.zip">
             <zipfileset dir="${dist}" includes="*.jar" prefix="picard-tools-${picard-version}"/>
-            <zipfileset dir="${htsjdk_lib_dir}" includes="htsjdk-*.jar" prefix="picard-tools-${picard-version}"/>
             <zipfileset dir="${lib}" includes="snappy*.jar"/>
-            <!-- distribute libJniDeflater.so in same directory as jarfiles-->
-            <zipfileset dir="${htsjdk}/lib/jni" includes="*" prefix="picard-tools-${picard-version}"/>
```

Comment:
Can you check whether `libJniDeflater.so` gets packaged in the Picard jar with the new build file? Currently htsjdk/Picard require that the path to the deflator be passed in as a system property, but shortly we'll be changing this so that it finds the deflator on the classpath and extracts and loads it at runtime.